src/inc/piece/tetris_piece.h

```diff
@@ -63,26 +63,16 @@ class TetrisPiece : public GameComponent
 
   std::vector<std::unique_ptr<Point>>& get_block_locations();
 
-  std::vector<std::unique_ptr<Point>> get_corners_to_check_left();
-
-  std::vector<std::unique_ptr<Point>> get_corners_to_check_right();
-
   void fall();
 
 protected:
 
   void set_block_locations( std::vector<std::unique_ptr<Point>> block_locations );
 
-  void set_corners_to_check( std::vector<std::unique_ptr<Point>> corners );
-
   virtual std::vector<std::unique_ptr<Point>> original_block_locations() = 0;
 
   virtual std::vector<std::unique_ptr<Point>> rotate_block_locations() = 0;
 
-  virtual std::vector<std::unique_ptr<Point>> original_corners() = 0;
```

Comment:
Checking these corners was an inelegant way to determine if a piece could shift one way or another. It was only necessary 'cause each piece was falling twice on a shift left or right. 

Removing these virtual methods forces the methods in the individual piece subclasses to be deleted as well.